.github/workflows/Helix-CI.yml

```diff
@@ -16,6 +16,8 @@ jobs:
       uses: actions/setup-java@v1
       with:
         java-version: 11
+    - name: Delete frontend-maven-plugin dir
```

Comment:
this can slow down the whole build process. Are there specific library which you want to delete?